### PR TITLE
Add support for community edition of couchbase server

### PR DESCRIPTION
**Pull request by Gerschtli**
Follow up for #4041.

### CODE REVIEW

**Review by daschl — APPROVED** [review]
Great work!

File: modules/couchbase/src/test/java/org/testcontainers/couchbase/CouchbaseContainerTest.java [hunk 1]
```diff
@@ -61,13 +64,36 @@ public void testBasicContainerUsage() {
         }
     }
 
+    @Test
+    public void testBasicContainerUsageForCommunityContainer() {
```
Comment (kiview) [inline modules/couchbase/src/test/java/org/testcontainers/couchbase/CouchbaseContainerTest.java:68]: I wonder why this new method is specifically necessary for the community container. Can't we make the original `testBasicContainerUsage` a parameterized test?

Comment (Gerschtli) [reply modules/couchbase/src/test/java/org/testcontainers/couchbase/CouchbaseContainerTest.java:68]: We could, but I don't know how to write parameterized tests in junit4 for only one test method.

**Comment by daschl** [comment]
@kiview I guess we can always refactor later, but right now the  number of tests is very limited so that little bit of duplication shouldn't be a problem.

File: modules/couchbase/src/test/java/org/testcontainers/couchbase/CouchbaseContainerTest.java (hunk 1 above)
Comment (kiview) [reply modules/couchbase/src/test/java/org/testcontainers/couchbase/CouchbaseContainerTest.java:68]: You are right, for this we would need to extract to its own class, so maybe not worth it.
However, why does the test itself differ from the `testBasicContainerUsageForEnterpriseContainer` body? Is it important to test other things explicitly for community container?

Comment (Gerschtli) [reply modules/couchbase/src/test/java/org/testcontainers/couchbase/CouchbaseContainerTest.java:68]: The only reason behind both tests is to check if the containers can be started and some smoke tests. Without my changes the Community edition failed to start.

Comment (kiview) [reply modules/couchbase/src/test/java/org/testcontainers/couchbase/CouchbaseContainerTest.java:68]: I was referring to the code as part of `setUpClient` in `testBasicContainerUsageForCommunityContainer`. This is not present in the `testBasicContainerUsageForEnterpriseContainer` test and I like to understand why the test differs in this way.

Comment (Gerschtli) [reply modules/couchbase/src/test/java/org/testcontainers/couchbase/CouchbaseContainerTest.java:68]: I am sorry, I don't get it. The tests of Community and Enterprise are exactly (and should be) the same except the different docker images.

**Review by dnault — APPROVED** [review]

File: modules/couchbase/src/test/java/org/testcontainers/couchbase/CouchbaseContainerTest.java (hunk 1 above)
Comment (kiview) [reply modules/couchbase/src/test/java/org/testcontainers/couchbase/CouchbaseContainerTest.java:68]: Sorry you are right, I misread the diff.

**Review by kiview — APPROVED** [review]

**Review by rnorth — APPROVED** [review]
I think this looks fine to me, thank you @Gerschtli!

**Pull request merged**